docs/sdpctl_appliance_backup.html

```diff
@@ -51,7 +51,7 @@ <h2 class="text-left margin-bottom">sdpctl appliance backup</h2><p>Perform backu
 <hr class="margin-bottom"><h3 class="emphasize text-left margin-bottom-small">Options</h3>
 <pre class="code-editor margin-bottom"><code>      --all                  backup all appliances in the Collective
       --current              backup the current peer Controller
-  -d, --destination string   backup destination directory (default &quot;$HOME/Downloads/appgate/backup&quot;)
+  -d, --destination string   backup destination directory (default &quot;$HOME/Scaricati/appgate/backup&quot;)
```

Comment:
Is that your user homedir? 😄

Reply:
Yeah… why is it dynamically using my env there? :D :D :D

Reply:
I'll fix the makefile

Reply:
Ok It's because the --help text is dynamic.